__test__/progressRing.spec.ts

```diff
@@ -0,0 +1,26 @@
+import '@testing-library/jest-dom/extend-expect';
+import 'jest';
+import { render, cleanup } from '@testing-library/svelte';
+import ProgressRing from './progressRing.svelte';
```

Comment:
test failed, change to `../progressRing.svelte`

Reply:
Resolved.